tests/e2e/Services/Databases/DatabasesBase.php

```diff
@@ -316,6 +316,23 @@ public function testCreateAttributes(array $data): array
         return $data;
     }
 
+    /**
+     * @depends testCreateAttributes
+     */
+    public function testListAttributes(array $data): void
+    {
+        $databaseId = $data['databaseId'];
+        $attributes = $this->client->call(Client::METHOD_GET, '/databases/' . $databaseId . '/collections/' . $data['moviesId'] . '/attributes', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey'],
+        ]), [
+            'queries' => ['equal("type", "string")'],
+        ]);
+        $this->assertEquals(200, $attributes['headers']['status-code']);
+        $this->assertEquals(4, $attributes['body']['total']);
```

Comment:
lets also add a test to make sure select queries are not allowed

Reply:
lets also try some other queries like limit , offset

Reply:
done this